guides/source/tuning_perf_for_deployment.md

```diff
@@ -0,0 +1,158 @@
+**DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
+
+Tuning Performance for Deployment
+=================================
+
+This guide covers performance and concurrency configuration for deploying your production Ruby on Rails application.
+
+After reading this guide, you will know:
+
+* Whether to use Puma, the default application server
+* How to configure important performance settings for Puma
+* How to begin performance testing your application settings
+
+More information about how to configure your application can be found in the [Configuration Guide](configuring.html).
+
+--------------------------------------------------------------------------------
+
+Choosing an Application Server
```

Comment:
Maybe in this section mention that Puma is the default one and that it should be fine for most cases?

Reply:
Okay. I've updated to address your feedback. I'll hit "resolve" for the changes I think are handled. Feel free to un-resolve or add comments if you feel I missed something.

Reply:
Of course! I'm ok, just trying to help :) Thanks for working in this so necessary guide 🙌

Reply:
No problem! This is really useful feedback and I appreciate it :-)